server/config/config.go

```diff
@@ -80,6 +80,14 @@ type Config struct {
 	LogFileDeprecated  string `toml:"log-file" json:"log-file,omitempty"`
 	LogLevelDeprecated string `toml:"log-level" json:"log-level,omitempty"`
 
+	// MaxConcurrentTSOProxyStreamings is the maximum number of concurrent TSO proxy streaming process routines allowed.
+	// Exceeding this limit will result in an error being returned to the client when a new client starts a TSO streaming.
+	// Set this to 0 will disable TSO Proxy.
+	// Set this to the negative value to disable the limit.
+	MaxConcurrentTSOProxyStreamings int `toml:"max-concurrent-tso-proxy-streamings" json:"max-concurrent-tso-proxy-streamings"`
+	// TSOProxyClientRecvTimeout is the timeout for the TSO proxy to receive a tso request from a client via grpc TSO stream.
+	TSOProxyClientRecvTimeout typeutil.Duration `toml:"tso-proxy-client-recv-timeout" json:"tso-proxy-client-recv-timeout"`
```

Comment:
It needs some additional steps to add new config. If no necessary, I think it is better to not add it.

Reply:
What's the additional steps? Xiaoguang reviewed this pr and asked for configuration. Do we have better way to add configuration?

Reply:
Per offline with @lhy1024 , we need a config review to review the config change, and @rleungx should be the right one.

Reply:
IMO, we don't need these configs since they will be replaced if we implement the discovery in other clients. Just using a constant is enough.

Reply:
I think we need if we can update config in the dev/staging/prod quickly without making code change, cherry-pick, hot fix and deployment.